compaction/compaction_manager.cc

```diff
@@ -742,21 +742,21 @@ void compaction_manager::submit_offstrategy(replica::table* t) {
                 _stats.active_tasks++;
                 task->setup_new_compaction();
 
-                return t->run_offstrategy_compaction(task->compaction_data).then_wrapped([this, task] (future<> f) mutable {
+                return t->run_offstrategy_compaction(task->compaction_data).then_wrapped([this, task, schema = t->schema()] (future<> f) mutable {
                     _stats.active_tasks--;
                     task->finish_compaction();
                     try {
                         f.get();
                         _stats.completed_tasks++;
                     } catch (sstables::compaction_stopped_exception& e) {
-                        cmlog.info("off-strategy compaction: {}", e.what());
+                        cmlog.info("off-strategy compaction of {}.{} was stopped: {}", schema->ks_name(), schema->cf_name(), e.what());
                     } catch (sstables::compaction_aborted_exception& e) {
                         _stats.errors++;
-                        cmlog.error("off-strategy compaction: {}", e.what());
+                        cmlog.info("off-strategy compaction of {}.{} was aborted: {}", schema->ks_name(), schema->cf_name(), e.what());
```

Comment:
what's the reason to switch from error to info? i assume it's because this is an user-triggered operation, but that's stopped exception no?

Reply:
Oops, cut-n-paste type.

Reply:
will fix.

Reply:
Fixed in v2